app/containers/Network/containers/AccessPointDetails/index.js

```diff
@@ -232,6 +242,30 @@ const AccessPointDetails = ({ locations }) => {
       );
   };
 
+  const handleDeleteEquipment = () => {
+    deleteEquipment({
+      variables: { id },
+      update: cache => {
+        Object.keys(cache.data.data).forEach(
+          key => key.match(`Equipment:${id}`) && cache.data.delete(key)
+        );
+      },
```

Comment:
remove this, instead add `fetchPolicy: 'network-only',` to the GET_EQUIPMENT query